pkg/render/common/elasticsearch/service.go

```diff
@@ -25,13 +25,26 @@ const (
 	linseedEndpoint     = "https://tigera-linseed.%s.svc"
 	httpsFQDNEndpoint   = "https://tigera-secure-es-gateway-http.%s.svc.%s:9200"
 	linseedFQDNEndpoint = "https://tigera-linseed.%s.svc.%s"
+
+	// Handle for multi tenant
+	guardianEndpoint       = "https://guardian.calico-system.svc"
+	guardinaFQDNEndpoint   = "https://guardian.calico-system.svc.%s"
```

Comment:
```suggestion
	guardianFQDNEndpoint   = "https://guardian.calico-system.svc.%s"
```

Reply:
I think we do have a helper for this already, though, so we probably don't need these variables: https://github.com/tigera/operator/blob/e072c811dd0e6c1111add2787a6c8c98ee42801d/pkg/render/guardian.go#L684-L686